recipe/deploy/writable.php

```diff
@@ -18,6 +18,9 @@
         return;
     }
 
+    //verify that directories exist
+    run("mkdir -vp $dirs");
```

Comment:
@yisraeldov 
It doesn't work. Need to move it under `cd('{{release_path}}');`.
Also please add test in test/recipe/CommonTest.php:testWriteable(). It requires just adding non-existed folder in  `writable_dirs` and checking existence after `deploy:writable`.

Do you have time for that? If not then I'll do it.